### PR TITLE
LRQA-71063, LRQA-71069 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRQA-71063
https://issues.liferay.com/browse/LRQA-71069

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
Hi @91nilo,
1. We usually add the ticket number in description to indicate what story/bug/regression the testcase automate for. That means the ticket number should be a story or a bug or a regression. e.g. [Login#LoginWithSpanishOnlyEnabledLocale](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase#L87).
[LRQA-41349](https://issues.liferay.com/browse/LRQA-41349) apparently not any type of those tickets. Actually those testcases are introduced by [LRQA-29297](https://issues.liferay.com/browse/LRQA-29297) and [LRQA-28935](https://issues.liferay.com/browse/LRQA-28935). 
In this situation, we can hardly find the original story/bug tickets. Thus, it's fine to add the description without the ticket numbers. e.g. [Login#DefaultPortalLandingPageSetByProperty](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase#L125)
2. As Gabor mentioned in [PR#339](https://github.com/gaborlovasdotcom/liferay-portal/pull/339#pullrequestreview-904416265), it's better to replace the word "Assert" with "Verify that... XY could happen." This way the description sounds more Business focused, and less technical.
3. Other tiny suggestions, I commented in the code.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 1]
```diff
@@ -19,58 +19,82 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could delete a Custom Export."
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:22]: e.g.
@description = "Admin user can delete the custom export process."

In this description, we focused on two personas that are admin user and export process. Not only mention one user or one process.

Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:22]: e.g.
@description = "Admin user can delete the custom export process."

In this description, we focused on two personas that are admin user and export process. Not only mention one user or one process.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 2]
```diff
@@ -19,58 +19,82 @@ definition {
...
+			LAR.deleteCurrentAndPreviousProcess(larFileName = "CustomExport");
+		}
 
-		AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1");
+		task ("Assert that the Custom Export was deleted") {
+			AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1");
+		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Site admin should be able to delete a Export on the Documents and Media Portlet."
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:40]: Test point is that admin user can delete the portlet export process. No need to mention what portlet we used in the testcase.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 3]
```diff
@@ -19,58 +19,82 @@ definition {
...
 	@priority = "4"
 	test DeletePortletExportProcess {
-		Navigator.openURL();
+		task ("Go to Content & Data --> Documents and Media and Export a Document and Media") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:43]: Another simpler way of Content & Data --> Documents and Media is DM admin.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 4]
```diff
@@ -19,58 +19,82 @@ definition {
...
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
 
-		LAR.exportPortlet(larFileName = "DMExport");
+			LAR.exportPortlet(larFileName = "DMExport");
+		}
 
-		Refresh();
+		task ("Delete the Export") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:53]: Prefer to append with process. e.g. Delete the portlet export process.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 5]
```diff
@@ -19,58 +19,82 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could delete a Custom Export."
 	@priority = "4"
 	test DeleteCustomExportProcess {
-		LAR.exportSiteCP(
-			larFileName = "CustomExport",
-			siteScopeName = "Guest");
+		task ("Create a Custom Export") {
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:25]: Here you export just export the LAR file of a Site with some custom parameters, I think you could rather use something like: "Export the LAR file of the default site"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 6]
```diff
@@ -19,58 +19,82 @@ definition {
...
+				larFileName = "DMExport");
+		}
 
-		AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS");
+		task ("Assert that the Export was deleted") {
+			AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS");
+		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could relaunch a Custom Export."
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:66]: No need to use uppercase letters in custom export. 
As this is a process you retry to do again, you should use the word "process" at the end of the sentence.

**Comment by 91nilo** [comment]
Thanks for the reviews, I did some changes based on them, but if there's something else that I could do to improve the code, just let me know.
Thanks!

CC: @JoyceWang08211 @weller0246  @gaborlovasdotcom

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
Hi @91nilo,
As this combines 2 separate tickets, I separated your commit into 2.

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:export-import:nocompile

**Comment by JoyceWang08211** [comment]
LGTM

**Review by JoyceWang08211 — APPROVED** [review]

**Comment by JoyceWang08211** [comment]
ci:test:export-import:nocompile

**Pull request closed without merging**